seeders/20180207062057-demo-users.js

```diff
@@ -1,8 +1,10 @@
+const bcrypt = require('bcryptjs');
+
 module.exports = {
   up: (queryInterface, Sequelize) => queryInterface.bulkInsert('users', [{
     userId: 1,
     userName: 'John_Doe',
-    password: 'password',
+    password: bcrypt.hashSync('password', 10),
```

Comment:
not sure if its ok to modify seeders, confirm and write new seeders if necessary.

Reply:
seeders are going to be generated at the time of migrations, I feel this can be modified